src/main/features/claimant-response/routes/confirmation.ts

```diff
@@ -17,6 +39,7 @@ export default express.Router()
         {
           claim: claim,
           confirmationDate: MomentFactory.currentDate(),
-          repaymentPlanOrigin: getRepaymentPlanOrigin(claim.settlement)
+          repaymentPlanOrigin: claim.settlement && getRepaymentPlanOrigin(claim.settlement),
+          hasAcceptedDefendantsPaymentIntention: hasAcceptedDefendantsPaymentIntention(claim)
```

Comment:
Variable name should be `paymentIntentionAccepted` so that it does not sound like method.